### DISCUSSION TITLE
🟩 Board

### DISCUSSION THREAD

**Discussion by ghost** [EddieHubCommunity/support#1531] [Poll] [answered]
How about a :green_square: board? We already have #good-first-issues and #first-timers channels on discord. How about a website that displays good first issues. People can post new issues and opportunities for green squares.

We can use express and socket.io. We can also use the GitHub API  for loading messages and other things.

Vote by 👍🏻 and 👎🏻

**Comment by ghost** [comment]
Just to add up to my idea:

https://github.com/search?q=comments%3A%3C20+label%3A%22good+first+issue%22+state%3Aopen&type=Issues

We can get the same data using graphql! We would just need a bunch of labels to look up with.

This way we can display all :green_square: treats instead of the only ones posted by the people!

**Comment by vinzvinci** [comment]
Good idea 👍

**Comment by Vyvy-vi** [comment, marked as answer]
I'm not sure if EddieHub should be making some specific site for this, since there are a few sites that already list out `good-first-issue` type issues(they're both auto-generated and sometimes vouched and accepted into the catalog)

Moreover, IMHO the advanced github search can also be of much use.
The purpose(afaik) of sharing the issues _personally_ on the discord server in the #good-first-issue channel is to get issues from the community, that are presented as good-first-issues, after consideration from the members.
This benefits the community in many ways-
- People new to contributing can contribute in repos that are owned or maintained by community members, and thus they can get a great first-time contribution experience.
- People in the community can get some contributors, more ideas and perhaps shout-out their project
- There is more generated interest for contributing to projects, after good contribution experiences at EddieHub

I'm not sure if the goal of the channels is to act as a central-hub of all first-timers issues.
(These are my 2 cents on the plan. *If you're planning on making this separately from EddieHub, then I think it's a great idea and I wish you the best of luck for this venture :) Feel free to ask any help or any ideas in chat from all of us)

**Comment by vinzvinci** [comment]
If there's any website that has a different category for every language that offers `good-first-issue`. That would be great.
